### ISSUE TITLE
Project status?

### ISSUE THREAD

**Issue by walkerke** [slu-openGIS/tidyseg#1]
Hey Chris- just checking in about the status of this project.  I’m working on some things that could make use of this package this summer and I’d be interested in contributing.  I think it would be great if eventually we could replicate what the pysal group is doing on the Python side: https://github.com/pysal/segregation

Let me know, thanks!

**Comment by chris-prener** [comment]
hey @walkerke - thanks for reaching out. This has been back burnered but I've got a grant that just kicked in that is going to cover some time to put into this. I'd welcome any / all contributions you'd like to make... want to give me a sense of what types of functionality you'd need for what you're doing this summer?

**Comment by walkerke** [comment]
@chris-prener in a nutshell - I'm [working on this project](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1739662) right now where I'm looking at ways that tidycensus can connect with other analytic pipelines.  One side of this will focus on visualization tools but the other will work on topics that could be covered in this package.  

I think the general idea that would be useful is to take a tidy census dataset (e.g. what is returned by default by `get_acs()`) and compute any of those segregation measures available in PySAL within a `%>%` pipeline.  I'll probably start working on this in June and would be happy to contribute code to the package so long as it aligns with your package goals.

**Comment by chris-prener** [comment]
That sounds great - congrats on the grant! 

My guess is our goals are pretty similar... the ultimate goal is to have a platform that works with long or wide data, as well as spatial measures of segregation that would be built on `sf` objects. I'd like something that is compatible with `group_by()` - imagining a long `get_acs()` output with multiple geographies.

My rough plan was to tackle the a-spatial measures (dissimilarity, Gini, etc) first.

**Comment by knaaptime** [comment]
you guys seen this yet?

https://jstatsoft.org/article/view/v089i07

**Comment by chris-prener** [comment]
Yes... just discussed it today (and your name came up, too... ears burning?)! 

It looks super cool but doesn't fit into my existing workflows...

**Comment by knaaptime** [comment]
ha! the fates conspire i guess. I was just trolling Kyle's github to see if there was new stuff I'd been missing and lo, he had forked this repo.

agreed on OasisR. we just discovered it last week

**Comment by chris-prener** [comment]
So did I @knaaptime. I feel bad for the author - it seems to have languished in peer review for several years. At any rate - we're planning some development work on this over the summer. Very inspired with the work ya'll are doing on the python side!

**Comment by knaaptime** [comment]
awesome, look foward to seeing it come together. keep in touch if there's anything we can do to help out. always happy to collaborate :)

**Comment by walkerke** [comment]
Hey @knaaptime!  Hope you're doing well.  I've been eagerly following all the new stuff that you've been working on at UCR.  _Almost_ enough to convince me to move my workflows over to Python... 😉 

I second everything @chris-prener mentioned.  The big thing is being able to do segregation analysis entirely within a tidyverse and sf-centric workflow.  Agree about collaboration - it's a natural fit.  We'll keep you posted on our progress!
